modules/tax/src/TaxTypeRepository.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\commerce_tax;
+
+use CommerceGuys\Tax\Repository\TaxTypeRepositoryInterface;
+
+/**
+ * Manages tax types based on JSON definitions.
```

Comment:
That's not true :) We should say something like "Exposes Drupal data to the tax library." Or use "entities" instead of "data".